test/e2e/run_passwd_test.go

```diff
@@ -58,4 +58,16 @@ var _ = Describe("Podman run passwd", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.LineInOutputContains("passwd")).To(BeTrue())
 	})
+
+	It("podman can run container without /etc/passwd", func() {
+		SkipIfRemote()
+		dockerfile := `FROM alpine
+RUN rm -f /etc/passwd /etc/shadow /etc/group`
```

Comment:
This test is missing the `USER 1000` command. Without `USER` in Dockerfile podman runs the image just fine.

Reply:
Good catch, fixed